cmd/gentypes/gentypes.go

```diff
@@ -107,7 +107,15 @@ func parsePropertyType(propValue map[string]interface{}) string {
 			if !ok {
 				log.Fatal("missing additionalProperties field when type=object:", propValue)
 			}
-			valueType := parsePropertyType(additionalProps.(map[string]interface{}))
+			var valueType string
```

Comment:
There is `additonalProperties: true` in `StartDebuggingRequestArguments.configuration`.
So I added type swtich to cover this situation.